src/main/java/com/kodilla/ecommercee/entity/UserOrder.java

```diff
@@ -19,4 +19,17 @@
     @Id
     @GeneratedValue
     private Long Id;
+
+    private LocalDate orderMade;
+
+    private boolean resolved;
+
+    @OneToOne(cascade = CascadeType.ALL,
+            fetch = FetchType.EAGER)
```

Comment:
Na pewno będzie tu relacja OneToOne? To by znaczyło, że jeden user może złożyć tylko jedno zamówienie.